Whiskey/Functions/Uninstall-WhiskeyPowerShellModule.ps1

```diff
@@ -50,9 +50,12 @@ function Uninstall-WhiskeyPowerShellModule
         }
     }
 
-    $psmodulesDirEmpty = $null -eq (Get-ChildItem -Path $modulesRoot -File -Recurse)
-    if( $psmodulesDirEmpty )
+    if( (Test-Path -Path $modulesRoot -PathType Container) )
     {
-        Remove-Item -Path $modulesRoot -Recurse -Force
+        $psmodulesDirEmpty = $null -eq (Get-ChildItem -Path $modulesRoot -File -Recurse)
+        if( $psmodulesDirEmpty )
+        {
+            Remove-Item -Path $modulesRoot -Recurse -Force

```

Comment:
We should have better discipline about when we use `-Path` vs `-LiteralPath`, otherwise we might run into a case where we accidentally delete a directory/file we didn't intend because `-Path` resolves wildcard characters.

Reply:
In this case, we want to use `Path` because we are using a `*.*.*` wildcard to remove all versions of a module.